### PR TITLE
LPS-156788 As a Sales Admin, I would like to view and modify the External Reference Code with the UI

### PR DESCRIPTION
**Pull request by RafaelCRC**

### CODE REVIEW

**Comment by RafaelCRC** [comment]
ci:test:sf

**Comment by RafaelCRC** [comment]
ci:test:relevant

File: modules/apps/account/account-api/src/main/java/com/liferay/account/exception/DuplicateAccountGroupExternalReferenceCodeException.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		super(msg, throwable);
+	}
+
+	public DuplicateAccountGroupExternalReferenceCodeException(
+		Throwable throwable) {
+
+		super(throwable);
+	}
+
+}
```
Comment (pei-jung) [inline modules/apps/account/account-api/src/main/java/com/liferay/account/exception/DuplicateAccountGroupExternalReferenceCodeException.java:44]: The exception classes are generated. To add a new exception, add it to the [service.xml](https://github.com/liferay/liferay-portal/blob/master/modules/apps/account/account-service/service.xml#L218-L230) and regen service builder.

**Review by pei-jung — CHANGES_REQUESTED** [review]
@RafaelCRC nice work! Just one comment for the exception.

**Comment by RafaelCRC** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:forward

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
@RafaelCRC can you fix the SF error? There's an SF issue with the exception class.

**Comment by RafaelCRC** [comment]
ci:test:sf

**Comment by RafaelCRC** [comment]
@pei-jung should be good to go now

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**